### ISSUE TITLE
Playback speed

### ISSUE THREAD

**Issue by Ivorytwelve** [asayake-b5/audiobooksync2renpy#1]
Hi,
Sometimes audiobooks tend to be pretty slow and I used to speed them up a little bit but now that I'm using this tool I can't anymore unfortunately.
I really like the tool and so I was wondering if adding a setting inside the AN to change the playback speed would be feasible. Thank you.

**Comment by asayake-b5** [comment]
Yeah it's something that I started contemplating doing recently as well, I had mostly used it on 本好き so far which had a good pace but some are so slow it borders on the snoozefest.

I kept  it on the back of my mind but I'll try to implement it in the coming days, it should be fairly easy to do but there's different solutions I thought of and I want to test out which one gives out the best result and will work cross-platform.

**Comment by Ivorytwelve** [comment]
Sweet, thank you.

I don't know if I should open another thread, so I'll just write it here, but here's some problems I found: 
1. Audio volume inside the AN is kinda low compared to the original audio of the audiobook
2. Sometimes the second half of the furigana of a word gets cut off like in here
![image](https://github.com/asayake-b5/audiobooksync2renpy/assets/118703097/5bce3cf6-b51f-4b95-92f1-c1212f4d4550)
3. Personally it's not a problem since I use the websocket but if I skip with the clipboard sender enabled I get this error:
![image](https://github.com/asayake-b5/audiobooksync2renpy/assets/118703097/e182de7a-04a2-41f7-a2bd-9afaa73e298d)

Also, I know it's still experimental, but for me, it doesn't show any of the images at all.

**Comment by asayake-b5** [comment]
> Audio volume inside the AN is kinda low compared to the original audio of the audiobook

That's weird, it should be closely similar, make sure your volume is full in the mixer, and in the renpy settings as well (preferences -> Voice Volume).  But I'll add a volume modifier as well just in case, that's not much of an issue.

> Sometimes the second half of the furigana of a word gets cut off like in here

Yeah there's almost as many handling of ruby as there are epubs, can you give the name of the audiobook/novel giving this problem so I can investigate?

> Also, I know it's still experimental, but for me, it doesn't show any of the images at all.

If you downloaded from the releases, that's normal, if you compiled from source, same as above, can you give me the audiobook name?

**Comment by Ivorytwelve** [comment]
> That's weird, it should be closely similar, make sure your volume is full in the mixer, and in the renpy settings as well (preferences -> Voice Volume). But I'll add a volume modifier as well just in case, that's not much of an issue.

I checked and I have both set at max volume

> Yeah there's almost as many handling of ruby as there are epubs, can you give the name of the audiobook/novel giving this problem so I can investigate?

The audiobook is オーバーロード1 不死者の王 前編 but also 後編

> If you downloaded from the releases, that's normal, if you compiled from source, same as above, can you give me the audiobook name?

Oh, my bad. I was indeed using the release version. I tried now compiling it but got an error that I can't seem to fix.
